test/properties.py

```diff
@@ -0,0 +1,284 @@
+        result = register_prop(self._new_prop, self._id)
+
+        self.assertEqual(result[0], ControllerStatus.SUCCESS)
+        self.assertEqual(PropertyDoc.objects.get(address=self._new_prop["address"]).address, self._new_prop["address"])
+
+
+class GetAllPropertiesTest(TestCase):
+    _email = "test@example.net"
+    _password = "testpass"
+    _id = ""
+    _register_user = {
+        "username": "testuser",
+        "passwd_hash": generate_password_hash(_password),
+        "email": _email,
+        "profile": {
+            "contact_email": _email
+        }
+    }
+    _register_prop = {
+        "address": "1234 Test Street",
+        "bathrooms_amount": 1.5,
+        "bedrooms_amount": 3,
+        "contract_type": "rent",
+        "currency_code": "EUR",
+        "floors_amount": 1,
+        "garage_size": 2,
+        "price": 120000,
+        "terrain_height": 5,
+        "terrain_width": 7
+    }
+
+    def setUp(self) -> None:
+        connect("fast-home-test", host="mongomock://localhost")
+        new_user = User(**self._register_user).save()
+        PropertyDoc(**self._register_prop).save()
+        self._id = str(new_user.id)
+
+    def tearDown(self) -> None:
+        disconnect()
+
+    def test_get_properties(self):
+        result = all_props()
+        obj = PropertyDoc.objects.get()
+
+        self.assertEqual(result[0], ControllerStatus.SUCCESS)
+        self.assertEqual(result[1].get(), obj)
+
+
+class UpdatePropertyTest(TestCase):
+    _email = "test@example.net"
+    _password = "testpass"
+    _userId = ""
+    _propId = ""
+    _register_user = {
+        "username": "testuser",
+        "passwd_hash": generate_password_hash(_password),
+        "email": _email,
+        "profile": {
+            "contact_email": _email
+        }
+    }
+    _register_prop = {
+        "address": "1234 Test Street",
+        "bathrooms_amount": 1.5,
+        "bedrooms_amount": 3,
+        "contract_type": "rent",
+        "currency_code": "EUR",
+        "floors_amount": 1,
+        "garage_size": 2,
+        "price": 120000,
+        "terrain_height": 5,
+        "terrain_width": 7
+    }
+
+    def setUp(self) -> None:
+        connect("fast-home-test", host="mongomock://localhost")
+        new_user = User(**self._register_user).save()
+        self._userId = str(new_user.id)
+        new_prop = PropertyDoc(**self._register_prop, owner=self._userId).save()
+        self._propId = str(new_prop.id)
+
+    def tearDown(self) -> None:
+        disconnect()
+
+    def test_successful_edit(self):
+        _price = 10
+        result = update_prop({
+            "price": _price,
+            "terrain_height": 2,
+            "terrain_width": 3,
+            "id": self._propId
+        }, self._userId)
+
+        self.assertEqual(result, ControllerStatus.SUCCESS)
+        # I'm not that sure about this last line, still passes
+        self.assertEqual(PropertyDoc.objects.get(owner=self._userId).price, _price)
```

Comment:
Preferiría que filtraras por el id de la propiedad, ya que esto daría error si hubiera varias propiedades con el mismo dueño, pero está bien
```suggestion
        self.assertEqual(PropertyDoc.objects.get(id=self._propId).price, _price)
```

Reply:
Nice. Creo que había puesto el owner para probar si funcionaba y como jaló no le moví xD. Grax por el comment